tools/drive-archaeologist/src/drive_archaeologist/archive_handler.py

```
@@ -69,11 +65,23 @@ def extract(self, filepath: Path) -> Optional[Path]:
                 with zipfile.ZipFile(filepath, 'r') as zf:
                     zf.extractall(temp_path)
```

Ruff (0.15.12)

[error] Uses of `tarfile.extractall()`

(S202)